armi/reactor/tests/test_components.py

```diff
@@ -497,47 +497,158 @@ def test_changeNumberDensities(self):
         self.component.changeNDensByFactor(3.0)
         self.assertEqual(self.component.getNumberDensity("NA23"), 3.0)
 
-    def test_amountConserved(self):
-        """Demonstrate that volume integrated ndense is conserved at different temperatures"""
+    def test_demonstrateWaysToExpand(self):
+        """Demonstrate that material is conserved at during expansion"""
+        ###########
+        #   # 1 2D Expansion
+        ###########
         # expansion only happens in 2D so only area is necessary
-        # since component expansion is only in 2D
-        tHotC = 20
-        circle1 = Circle("circle", "HT9", 20, tHotC, 1.0)
+
+        # when comparing to 3D density, the comparison is not quite correct.
+        # We need a bigger delta, this will be investigated/fixed in another PR
+        biggerDelta = 0.001  # g/cc
```

Comment:
The fact that we need this is indicative of a bug. Thankfully the differences are very minor.
https://github.com/terrapower/armi/issues/820

Reply:
To my understanding, if we expand in 2D, we use `Material.density()` and if we expand in 3D, we use `Material.density3()` which will then be the same as 2D density.

So, it appears to me that for the 2D example here, we could be seeing that.  It's the 3D expansion below that gives me pause.